src/e2e/specs/dashboard.spec.ts

```diff
@@ -642,3 +718,42 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, (
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Data Breaches`, () => {
+  test.beforeEach(async ({ landingPage, page, authPage }) => {
+    await forceLoginAs(
+      "joe@mailinator.com",
+      "TestPass1234",
+      page,
+      landingPage,
+      authPage,
+    );
+  });
+
+  test("Verify that the High risk data breaches step is displayed correctly", async ({
+    dashboardPage,
+    dataBrokersPage,
+    page,
+  }) => {
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463592",
+    });
+
+    await expect(dashboardPage.upsellScreenButton).toBeVisible();
+    await dashboardPage.upsellScreenButton.click();
+    await page.waitForURL(/.*\/data-broker-profiles\/view-data-brokers\/?/);
+    await expect(dataBrokersPage.forwardArrowButton).toBeVisible();
+    await dataBrokersPage.forwardArrowButton.click();
+    await page.waitForURL(/.*\/high-risk-data-breaches.*/);
+    const highRiskDataBreachLi = page.locator(
+      'li:has(div:has-text("High risk data breaches"))',
+    );
```

Comment:
I'm confused on this locator vs the ... ah, so _this_ locator just matches an `<li>` but it's the sub-sub-locator on L756 that actually gets the child div's text content.

Reply:
Yeah, so its something like: find a li that has a div that has a text.